.minipupper.repos

```diff
@@ -0,0 +1,13 @@
+repositories:
+  champ/champ:
+    type: git
+    url: https://github.com/chvmp/champ.git
+    version: ros2
+  champ/champ_teleop:
+    type: git
+    url: https://github.com/chvmp/champ_teleop.git
+    version: ros2
+  ldlidar_stl_ros:
+    type: git
+    url: https://github.com/ldrobotSensorTeam/ldlidar_stl_ros2.git
+    version: ros2
```

Comment:
It doesn't exist anymore. They have only `master` branch at the moment.

Reply:
Thanks, fixed. [5b4c6b0](https://github.com/mangdangroboticsclub/mini_pupper_ros/pull/40/commits/5b4c6b0e3632b5c1806a97836754d0e11c2afb50)